releasenotes/notes/update-openssl-3-0-12-4c33d434e0822b48.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    Update OpenSSL from 3.0.11 to 3.0.12.
+    This addresses CVE-2023-5363
```

Comment:
```suggestion
    This addresses CVE-2023-5363.
```

Reply:
Applied manually while removing the omnibus commit to avoid spawning too many pipelines